contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcViews.java

```diff
@@ -102,13 +122,35 @@
           RpcViewConstants.RPC_SERVER_STARTED_COUNT_HOUR_VIEW,
           RpcViewConstants.RPC_SERVER_FINISHED_COUNT_HOUR_VIEW);
 
+  /**
+   * Registers all standard gRPC views.
+   *
+   * <p>It is recommended to call this method before doing any RPC call to avoid missing stats.
+   *
+   * @since 0.13
+   */
+  public static void registerAllGrpcViews() {
+    registerAllCumulativeViews(Stats.getViewManager());
```

Comment:
Shouldn't this call registerAllGrpcViews(ViewManager)?

Reply:
That's right, done.